Makefile

```
@@ -200,7 +200,14 @@ lint:
 	kubeval ./k8s/*
 
 test:
-	./tools/test -netgo
+	go test -tags netgo -timeout 30m -race -count 1 ./...
```

You missed `-covermode=atomic` which was specified in `tools/test`.

```suggestion
	go test -tags netgo -timeout 30m -race -covermode=atomic -count 1 ./...
```

---

I don't think we need the covermode here because we're not doing coverage in the test target and if we set `covermode` we're enabling the cover part for the tests as you can see [here](https://github.com/golang/go/blob/06ac303f6a14b133254f757e54599c48e3c2a4ad/src/cmd/go/internal/test/test.go#L229)

On the other hand in the cover target is being used because is the default mode in case we're using race as you can see [here](https://github.com/golang/go/blob/06ac303f6a14b133254f757e54599c48e3c2a4ad/src/cmd/go/internal/test/test.go#L222-L223)

---

Oh you're right. All good here, thanks!